File: src/main/java/dev/shota/decompiler/window/menu/file/File.java
package dev.shota.decompiler.window.menu.file;

import dev.shota.decompiler.window.utils.Language;
import javax.swing.*;
import java.awt.event.KeyEvent;

public class File extends JMenu {

    public File() {
        setText(Language.get("file"));
        setMnemonic(KeyEvent.VK_F);

        add(new FileExit());
    }

}

File: src/main/kotlin/dev/shota/decompiler/window/menu/file/File.kt
package dev.shota.decompiler.window.menu.file

import dev.shota.decompiler.window.menu.Menu
import dev.shota.decompiler.window.menu.file.items.*
import java.awt.event.KeyEvent
import javax.swing.JSeparator

class File : Menu("file", KeyEvent.VK_F) {

    init {
        add(OpenFile())
        add(OpenProcess())
        add(CloseTab())
        add(JSeparator())
        add(NewWindow())
        add(Exit())
    }

}